skimage/future/graph/rag.py

```diff
@@ -51,21 +41,79 @@ def min_weight(graph, src, dst, n):
     return min(w1, w2)
 
 
+def _add_edge_filter(values, graph):
+    """Create edge in `g` between the first element of `values` and the rest.
+
+    Add an edge between the first element in `values` and
```

Comment:
I think this docstring should be updated ? Since you made the middle value as the center